Dockerfile

```diff
@@ -1,18 +1,16 @@
 FROM node:12.17.0-alpine
 
-ENV YARN_CACHE_FOLDER=/dev/shm/yarn-cache
-
 WORKDIR /app
 
 RUN apk --no-cache add --virtual native-deps \
   g++ gcc libgcc libstdc++ linux-headers make python git \
   chromium chromium-chromedriver
 
-COPY yarn.lock package.json /app/
-RUN yarn --dev --frozen-lockfile
+COPY package*.json /app/
+RUN npm install --development
 
 COPY . /app/
 
 EXPOSE 3001
 
-CMD ["yarn", "start"]
+CMD ["npm", "start"]
```

Comment:
any reason why we use `ts-node` instead of building it and starting the dist directly with `node`?

Reply:
I have the same question as you. `ts-node` is just `node` + ´tsc´. I think it's better to use the native node binary instead of `ts-node` to run user service. `ts-node` has increased memory usage compared to `node`, and it's also slower than `node`. Also, the bridge between build- and run-time is not that clear with `ts-node`. Better to first build the project with `tsc`, which will emit errors straight away.

Reply:
yeah + fewer deps to include in the output image

Reply:
@MergHQ any reason for ts-node over tsc at image build time?

Reply:
Emmä vittu tiedä, kai laiskuus on se syy

Reply:
iha ok

Reply:
Mut hyvä idea olla käyttämättä ts-nodee prodissa